agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -94,8 +94,18 @@ def evaluate(
     loop = asyncio.get_event_loop()
 
     try:
-        # 1. Fetch data from the database
         loop.run_until_complete(DBEngine().init_db())
+
+        # 0. Update evaluation status to STARTED
+        loop.run_until_complete(
+            update_evaluation(
+                evaluation_id,
+                {"status": Result(type="status", value="EVALUATION_STARTED")},
```

Comment:
We should use `EvaluationStatusEnum.EVALUATION_STARTED` here and not hard code the value

Reply:
True, fixed.